templates/examples/docker_jenkins/startup.sh

```diff
@@ -0,0 +1,6 @@
+# cloning mdbci repo
+git clone https://github.com/OSLL/mdbci.git
+
+# Running jenkins
+java -jar jenkins.war --httpPort=5555
```

Comment:
It is possible, we need only wget some .hpi file and put them in jenkins plugin directory.